CONTRIBUTING.md

```diff
@@ -1,80 +1,99 @@
 # Contributor guidelines
 
-We'd love to accept your patches and contributions to this project. There are just a few guidelines to follow.
+We are excited that you're interested in contributing to our project! Your efforts help us improve and grow.
 
-When contributing to this project, you must agree that you have authored the content, that you have the necessary rights to the content and that the content you contribute may be provided under the [APACHE LICENSE, VERSION 2.0](LICENSE).
+## Code of conduct
 
-### What do I need to know to help?
+This project adheres to a [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to abuse@score.dev.
 
-If you are looking to help to with a code contribution, our project uses Go Lang. If you don't feel ready to make a code contribution yet, no problem! You can also check out the documentation <https://github.com/score-spec/docs>.
+## Ways to contribute
```

Comment:
Can we include a note about our code licensing? Why we have Apache, and why some repos are licensed to Humanitec?

Can this change with CNCF donation?